### PR TITLE
Feat: add functionality in video categories

### PR DESCRIPTION
**Pull request by Divya70**
Feat: add functionality in videos categories

image

image

### CODE REVIEW

**Review by Asuraking1n — APPROVED** [review]
You can use less codes. Otherwise good

File: src/Reducer/Video-Reducer.js [hunk 1]
```diff
@@ -5,6 +5,66 @@ const videoReduce = (state, action) => {
         ...state,
         initialVideo: action.payload,
       };
+    case "ALL":
```
Comment (Ayushi20-19) [inline src/Reducer/Video-Reducer.js:8]: this code be extracted more no need to write all the categories again, just make true or false by checking that one

File: src/components/VideoCategory/VideoCategory.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+  const { dispatch } = useVideo();
   return (
     <>
       <div className="fruit-category-container">
-        <button>All</button>
-        <button>Fashion</button>
-        <button>Study</button>
-        <button>Music</button>
-        <button>Food</button>
+        <button onClick={() => dispatch({ type: "ALL" })}>All</button>
```
Comment (Ayushi20-19) [inline src/components/VideoCategory/VideoCategory.js:9]: this can also be extracted by mapping the categories

Comment (Divya70) [reply src/components/VideoCategory/VideoCategory.js:9]: i will improve that later

**Review by Ayushi20-19 — APPROVED** [review]
Suggested some changes.

**Pull request merged**